### PR TITLE
metrics: Add cassandra benchmark

### LINKED ISSUE #4709: add cassandra k8s test

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds the cassandra benchmark for metrics using k8s.

Fixes #4709

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Review by dborquez — APPROVED** [review]
lgtm, thanks @GabyCT

File: metrics/disk/cassandra_kubernetes/cassandra.sh [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	export write_cmd="/usr/local/apache-cassandra-3.11.2/tools/bin/cassandra-stress write n=1000000 cl=one -mode native cql3 -schema keyspace="keyspace1" -pop seq=1..1000000 -node cassandra"
+ 	number_of_retries="50"
+	for _ in $(seq 1 "$number_of_retries"); do
+		if kubectl exec -i cassandra-0 -- sh -c 'nodetool status' | grep Up; then
+			ok="1"
+ 			break;
+		fi
+ 		sleep 1
+	done
+	sleep 30
```
Comment (chavafg) [inline metrics/disk/cassandra_kubernetes/cassandra.sh:36]: could you document what is this `sleep` for? If this is for having cassandra up, isn't the loop above enough?

**Review by chavafg — COMMENTED** [review]
looks good, just one minor comment

File: metrics/disk/cassandra_kubernetes/cassandra.sh (hunk 1 above)
Comment (GabyCT) [reply metrics/disk/cassandra_kubernetes/cassandra.sh:36]: @chavafg yes we need cassandra up, the loop is not enough to verify that

Comment (GabyCT) [reply metrics/disk/cassandra_kubernetes/cassandra.sh:37]: @chavafg changes applied

**Review by chavafg — APPROVED** [review]

**Pull request merged**